webserver/src/pages/Home/HomeSection.tsx

```diff
@@ -1,36 +1,41 @@
-import {lazy, useEffect, useState} from "react";
+import { lazy, useEffect, useState } from "react";
 
-import {Header} from '../../layouts/Header';
-import {Information} from '../../components/Information/Information';
+import { Header } from '../../layouts/Header';
+import { Information } from '../../components/Information/Information.tsx';
 
+// @ts-ignore
 import privateer_module from "../../wasm/privateer.js"
-import loadGlytoucan from "../../utils/loadGlytoucan"
+import loadGlytoucan from "../../utils/loadGlytoucan.js"
 
-const Footer = lazy(() => import('../../layouts/Footer'));
-const BorderElement = lazy(() => import('../../layouts/BorderElement'));
+const Footer = lazy(() => import('../../layouts/Footer.js'));
+const BorderElement = lazy(() => import('../../layouts/BorderElement.js'));
+
+import { fetch_map, fetch_pdb } from "../../utils/fetch_from_pdb.js"
```

Comment:
Why do these imports have `.js` ending instead of `.ts` when the file itself has been refactored to `utils/fetch_from_pdb.ts` ?